### PR TITLE
DOC-5425: Updated CSDE port 7000 to 5005

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ This section lists the third-party software prerequisites for local deployment w
 For information about the prerequisites for multi-worker cluster deployments, see [Prerequisites for Cluster Deployments](../../../deploying/prerequisites.html).
 
 {{< note >}}
-You cannot start Corda via the CSDE `startCorda` task if any existing local programs are using ports 5432, 7000, or 8888. Reserve these ports.
+You cannot start Corda via the CSDE `startCorda` task if any existing local programs are using ports 5432, 5005
+, or 8888. Reserve these ports.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md:17]:
```suggestion
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md [hunk 2]
```diff
@@ -13,7 +13,8 @@ This section lists the third-party software prerequisites for local deployment w
 For information about the prerequisites for multi-worker cluster deployments, see [Prerequisites for Cluster Deployments](../../../deploying/prerequisites.html).
 
 {{< note >}}
-You cannot start Corda via the CSDE `startCorda` task if any existing local programs are using ports 5432, 7000, or 8888. Reserve these ports.
+You cannot start Corda via the CSDE `startCorda` task if any existing local programs are using ports 5432, 5005
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md:16]:
```suggestion
You cannot start Corda via the CSDE `startCorda` task if any existing local programs are using ports 5432, 5005, or 8888. Reserve these ports.
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md [hunk 3]
```diff
@@ -45,11 +46,11 @@ This what is known to work with the code as of Developer Preview 2.-->
 
 ## Required CSDE Ports
 
-The current version of the [Corda combined worker](../cordapp-standard-development-environment/csde.html#gradle-helpers-for-the-combined-worker) runs on ports 7000, 5432, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 7000 for the system Control Center. The following section describes how to make the port available.
+The current version of the [Corda combined worker](../cordapp-standard-development-environment/csde.html#gradle-helpers-for-the-combined-worker) runs on ports 5005, 5432, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 5005 for the system Control Center. The following section describes how to make the port available.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md:49]: Is it definitely true that the port number for the system Control Center has changed?

**Review by nadinequinr3 — DISMISSED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md (hunk 3 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0-beta/developing/getting-started/prerequisites/prerequisites.md:48]: Confirmed with @manishr3 that the system Control Center port number has changed to 5005.

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**